koku/sources/sources_error_message.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, error):
     def azure_client_errors(self, message):
         """Azure client error messages."""
         scrubbed_message = ProviderErrors.AZURE_GENERAL_CLIENT_ERROR_MESSAGE
-        if any(test in message for test in ["http error: 401", "Authentication failed"]):
+        if any(test in message for test in ["http error: 401", "Authentication failed", "(401) Unauthorized"]):
```

Comment:
This inline string is getting a bit long. One more it's best to move this to a constant.